### DISCUSSION TITLE
Convert type to array, if "->multiple()" is set but only 1 element selected

### DISCUSSION THREAD

**Discussion by Nestoran** [awcodes/filament-curator#219] [General]
Currently if the "->multiple()" setting is set, but only 1 media is selected, then the result is integer.

This is inconvenient fo example when used in a @foreach() cycle, where error is thrown if the value is not an array.

We must manually mutate the data to convert the integer into an array, even though we already expect to work with an array, since we declared that it could be "->multiple()".

**Comment by awcodes** [comment]
This will be fixed in the next release. Thanks.

**Comment by Nestoran** [comment]
Has this been fixed? I am still experiencing this issue in v2.10.1.

**Comment by awcodes** [comment]
https://github.com/awcodes/filament-curator/commit/8d674ea96911dd95dc9a25c8d07ecd80691fb8a0
